EDILibrary/EdifactFormatVersion.cs

```diff
@@ -335,7 +345,7 @@ public class EdifactFormatVersionHelper : IEdifactFormatVersionProvider
         /// <summary>
         /// validity date of <see cref="EdifactFormatVersion.FV2310"/>
         /// </summary>
-        private static readonly DateTime KeyDate2310 = new(2023, 09, 30, 22, 0, 0, DateTimeKind.Utc);
+        private static readonly DateTime KeyDate2310 = new(2023, 09, 30, 23, 0, 0, DateTimeKind.Utc);
```

Comment:
```suggestion
        private static readonly DateTime KeyDate2310 = new(2023, 09, 30, 22, 0, 0, DateTimeKind.Utc);
```
https://www.zeitumstellung.in/deutschland-2023/

Reply:
hupsi, danke